STEERING-COMMITTEE.md

```diff
@@ -48,6 +48,37 @@ the way we run this committee, based on feedback from the community.
 1. Act as the final escalation point and decider for any disputes, issues,
    clarifications, or escalations within the project scope.
 
+## Knative Trademark Committee Charter
+
+1. Manage licensing the Knative brand and decide which things can be called
+   "Knative" and how that mark can be used in relation to other efforts or
+   vendors.
+1. Review and approve conformance rules and tests that define brand use
+   decisions.
+1. Review and approve definition of Knative Core scope.
+1. Only the Trademark Committee can redefine its own charter and authority over
+   Knative Trademark, conformance, and brand decisions.
+
+## Trademark Committee Composition
+
+The Trademark Committee is composed of company representatives from companies
+which have substantially contributed to Knative over the last year. For the 2020
+year, one seat is assigned to each of Google, IBM/RedHat, and VMware. The
+committee shall review this composition at least annually and update these
+rules.
+
+The Trademark Committee desires to always reach consensus. Additionally, each
+member can veto decisions regarding conformance and scope.
+
+Decisions are made in meetings when a quorum of the members are present and may
+pass with at least half the members of the committee supporting it.
+
+Quorum is considered reached when at least of the members are present.
```

Comment:
If all decisions are by consensus, then where's the need for a quorum?

Reply:
I like the idea of allowing abstinence votes. This allows a lot of progress. The US Declaration of Independence, famously if I recall, would not have passed without this... Allowing no votes gives some flexibility to some folks. Of course, we are talking about a tiny group here so perhaps this is more theory than practice.

Reply:
Sure.  On the other hand, if you have an absolute voting rule ... e.g. "a majority of the KTC must approve all decisions", that makes it fairly simple to schedule meetings and even have emergency votes via email.  And you don't have to worry about quorums.  Also, as you point out, with an initial 3 members, there's not much point in having quorum rules, given.

Reply:
I added the ability of non-present members to veto a decision for 5 business days, which seems like it should avoid "quorum games".